app/views/widgets/radioBoxHorizontal.scala.html

```diff
@@ -0,0 +1,13 @@
+@(field : play.api.data.Field, options: Seq[(String, String, Boolean)], optionTrigger: String = "", isCompound: Boolean = false)
+
+@import helpers.CSRSkinnyFieldConstructor._
+
+@createId(id:String, v:(String,String,Boolean))=@{id + "-"+v._1.replace(" ", "_").replace("/", "_")}
+
+@helper.input(field, '_class -> (if(isCompound) "form-group-compound" else "inline")) { (id, name, value, args) =>
+    @for(v <- options){
```

Comment:
this code might be clear if you destructure the tuple in the for comprehension. That would remove the need to reference the fields by position further down